2023-round-1/devotuoma/nx_version.txt

```diff
@@ -0,0 +1,2 @@
+2.7.1
+
```

Comment:
thanks for this @devotuoma!

Could you try upgrading your python version and networkx version?